### DISCUSSION TITLE
Does NativeScript support SiriKit?

### DISCUSSION THREAD

**Discussion by Xaypanya** [NativeScript/NativeScript#10613] [Q&A]
How can I integrate SiriKit into a NativeScript app to handle voice commands like 'Perform action X for amount Y'? Any examples or pointers to existing plugins would be greatly appreciated!

**Comment by NathanWalker** [comment]
Would make for a great example, it does. You have to integrate activity types into plist so it takes some configuration. Will add this on a TODO to create blog post, and maybe a Snack could be shared, runnable from stackblitz.
